src/sdk/pynni/nni/nas/tensorflow/graph_utils.py

```diff
@@ -0,0 +1,249 @@
+import functools
+import weakref
+
+import tensorflow as tf
+from tensorflow.python.eager import function as function_lib
+from tensorflow.python.framework import func_graph, ops
+from tensorflow.python.ops import array_ops, control_flow_ops, math_ops, resource_variable_ops
+from tensorflow.python.util import object_identity, tf_decorator
+
+from . import lift_to_graph
+
+class Function:
```

Comment:
Should we add disclaimer to this code and comments on why this snippet of code should be copied and modified?